src/components/ModelDialog/components/ModalContent/styled.ts

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2023 Gemeente Amsterdam
+import { themeSpacing } from '@amsterdam/asc-ui'
+import styled from 'styled-components'
+
+import FormFooter from '../../../FormFooter'
+
+export const StyledSection = styled.div`
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  margin: ${themeSpacing(4)};
+`
+export const StyledDescription = styled.div`
+  padding-bottom: 56px;
+`
+
+export const StyledFormFooter = styled(FormFooter)`
+  box-sizing: border-box;
+  bottom: 0;
+  position: fixed;
+  height: 72px;
```

Comment:
Magic number. Does not make sense when you do no know the context

Reply:
Resolved it